src/usr/local/www/interfaces_assign.php

```diff
@@ -501,6 +536,32 @@ function interface_assign_description($portinfo, $portname) {
 $tab_array[] = array(gettext("Bridges"), false, "interfaces_bridge.php");
 $tab_array[] = array(gettext("LAGGs"), false, "interfaces_lagg.php");
 display_top_tabs($tab_array);
+//$timea = microtime(true);
+
+/*generate the port select box only once. 
+Not indenting the HTML and keeping each option to one line in
+this function results in HTML code that is 8.9KB smaller with 500
+VLANS than the original code structure. Multiplied by 500 VLANs 
+this means the page is ~4.5MB smaller and takes over 1s less to 
+transmit and also renders significantly faster in the browser.
+This is too much of an improvement to ignore. Tested with 500
+VLANS, total page output is 17,197.33KB with 8456ms spent 
+waiting for page generation and transmission, compared to 
+21,697.89KB with a wait of 9734ms*/
```

Comment:
This is a self-congratulatory comment.  "Keep HTML options on one line to minimize size/load-time" would suffice.